dev/com.ibm.ws.rest.handler.config/resources/com/ibm/ws/rest/handler/config/resources/CWWKOMessages.nlsprops

```diff
@@ -0,0 +1,33 @@
+###############################################################################
+# Copyright (c) 2019 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#CMVCPATHNAME com.ibm.ws.rest.handler.config/resources/com/ibm/ws/rest/handler/config/resources/CWWKOMessages.nlsprops
+#ISMESSAGEFILE TRUE
+#NLS_ENCODING=UNICODE
+#
+#COMPONENTPREFIX CWWKO
+#COMPONENTNAMEFOR CWWKO Config REST endpoint
+#
+# NLS_MESSAGEFORMAT_VAR
+#
+#   Strings in this file which contain replacement variables are processed by the MessageFormat 
+#   class (single quote must be coded as 2 consecutive single quotes ''). Strings in this file 
+#   which do NOT contain replacement variables are NOT processed by the MessageFormat class 
+#   (single quote must be coded as one single quote '). 
+
+# All messages must use the range CWWKO1530 to CWWKO1549
+
+CWWKO1530_CIRCULAR_DEPENDENCY=CWWKO1530I: Configuration information for {0} is truncated because it contains a cycle.
```

Comment:
Contains a bike?  Use of the word cycle here is ambiguous or incomplete.  Consider instead "contains a cyclic dependency"